src/controllers/historycontroller.js

```diff
@@ -1,47 +1,46 @@
 class HistoryController extends Controller
 {
-    static addEvent(event)
+    view()
     {
-        if (event instanceof Customer)
-        {
-            console.log('event is customer');
-
-            // this._loadTemplate(
-            //     "src/views/template/history/customer.html",
-            //     "#history-events",
-            //     event,
-            //     true
-            // );
-            //
-            // this.addCustomer(event);
+        this._loadTemplate(
+            "src/views/template/history.html",
+            "#history",
+            {}
+        );
+    }
 
-            /**
-             * The above method calls are not working,
-             * I get this error:
-             *
-             * Uncaught TypeError: this._loadTemplate is not
-             * a function at Function.addEvent
-             * (http://localhost:8888/Serious-Game-for-Kids/src/controllers/historycontroller.js:9:18)at <anonymous>:1:19
-             */
+    log(entry)
+    {
+        if (entry instanceof Customer)
```

Comment:
I do not think this warrants two different methods. You're essentially doing the same thing in both, just using different templates. I'd refactor `addContainer()` and `addCustomer()` into `log(entry)`, as they are not specific enough.